tests/test_inventory.py

```diff
@@ -318,12 +318,14 @@ def test_inventory_decay(self):
         """
 
         inv = Inventory({"H-3": 10.0})
-        self.assertEqual(inv.decay(12.32, "y").contents, {"H-3": 5.0})
+        self.assertEqual(inv.decay(12.32, "y").contents, {"H-3": 5.0, "He-3": 0.0})
         inv = Inventory({"Tc-99m": 2.3, "I-123": 5.8})
         self.assertEqual(
             inv.decay(20.0, "h").contents,
             {
                 "I-123": 2.040459244534774,
+                'Ru-99': 0.0,
+                'Sb-123': 0.0,
```

Comment:
If you try running [Black](https://black.readthedocs.io/en/stable/index.html) on the code, it will convert all these single quotes to double quotes. This is not something that is so important, but Black performs other helpful formatting to make the code more readable. I'll put a note in about this when I write a contributor guidelines section for the docs.